tests/test_main.py

```diff
@@ -1,3 +1,85 @@
 from app.main import validate_email
 
-# write your tests here
+import pytest
+
+
+@pytest.mark.parametrize(
+    "email, expected",
+    [
+        ("mate....academy.@mate.com", False),  # 1-st bug
+        (".d.e.v.my..email@dev.com", False),
+        ("d.e.v.my..email@dev.com.", False),
+        ("d.e.v.my.email@.dev.com", False),  # 2-nd bug
+        ("d.e.v.my.email@dev.com", True),
+    ],
+)
+def test_email_with_dots(email, expected):
+    assert validate_email(email) is expected
+
+
+@pytest.mark.parametrize(
+    "email, expected",
+    [
+        ("m#te_Ac^43*my@mate.com", False),
+        ("!@#$%^&*+my@dev.com", False),
+    ],
+)
+def test_email_with_special_symbols(email, expected):
+    assert validate_email(email) is expected
+
+
+@pytest.mark.parametrize(
+    "email, expected",
+    [
+        ("123456897@mate.com", True),
+        ("9876543210@dev.com", True),
+    ],
+)
+def test_email_only_with_digits(email, expected):
+    assert validate_email(email) is expected
+
+
+@pytest.mark.parametrize(
+    "email, expected",
+    [
+        ("agfdsayufgsa@mate.com", True),
+        ("DUYVSBFDSVFU@dev.com", True),
+        ("DUYVSBFDSVFUighdfsiufds@dev.com", True),
+    ],
+)
+def test_email_only_with_letters(email, expected):
+    assert validate_email(email) is expected
+
+
+@pytest.mark.parametrize(
+    "email, expected",
+    [
+        ("my_Post_is_corRect@mate.com", True),
```

Comment:
If here always `True` - then you can not add it here - just write it directly in the test :)
Also you can remake this to 1 big parameterized test with pytest.param and id - but it is not obligatory.

Reply:
okay, thank you for the advice about one parametrize with id, it's a good idea